scripts/examples-diff.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# 1. Compile all examples to _output
+dir="examples/_output"
+. scripts/examples-compile.sh
+
+diffdir="examples/_diff"
+mkdir $diffdir
+rm -f $diffdir/*
+
+# 2. Diff all examples
+changed=() # array for collecting changed examples
+for file in examples/*.json; do
+  name=${file##*/} # filename only (no dir path)
+  base=${name%.json} # exclude extension
+  if [ "$name" != "vlexamples.json" ]; then  # don't compile the list json
+    diff=$(json-diff --color examples/_original/$name examples/_output/$name)
+    if [ "$diff" != " undefined" ]; then
+      echo "Diff for $name"
+      echo -e "$diff"
+      changed+=("$base")
+      # compile each vega output to svg in examples/_diff
+      # and use "xmllint --format -" to format
```

Comment:
Umm, most of the time, you should just compare rendered svg.  (Mac’s quicklook supports SVG I think)